### ISSUE TITLE
[1.11.2] zero size item stack in grid caused Null pointer in .GridStackItem.getTooltip(GridStackItem.java:79)

### ISSUE THREAD

**Issue by PrinceOfAmber** [raoulvdberge/refinedstorage#1126]
#### Issue description:
Have a working system with a grid and disk drive

#### What happens:
Added and removed some mods, then noticed an empty slot with a 'zero' on it, so looked like a blank space but with number 0 size.  Clicking on it caused crash

#### What you expected to happen:
i expect not to see any item stacks of size zero.

#### Steps to reproduce:
1.Have a working system with a grid and disk drive
2., then noticed an empty slot with a 'zero' on it, 
3..  Clicking on it caused crash
...

#### Version (Make sure you are on the latest version before reporting):

- Minecraft: 1.11.2
- Forge: 13.20.0.2259
- Refined Storage: 1.4.2

Does this issue occur on a server? [yes/no] no

#### If a (crash)log is relevant for this issue, link it here:

https://pastebin.com/CSwgrKZz

**Comment by raoulvdberge** [comment]
What storage is causing this zero stack? Can you narrow this down for me to a single reproduceable case?

**Comment by Dykam** [comment]
I've just experienced a similar issue, there is an empty air itemstack stored. Hovering (tooltip) worked fine, however searching causing only the empty stack to be left caused a crash: `OreDictionary.getOreIDs` "Stack can not be invalid!".

https://github.com/raoulvdberge/refinedstorage/blob/aba639c/src/main/java/com/raoulvdberge/refinedstorage/gui/grid/stack/GridStackItem.java#L72 -> https://github.com/MinecraftForge/MinecraftForge/blob/04885b3  /src/main/java/net/minecraftforge/oredict/OreDictionary.java#L440

Crashlogs https://gist.github.com/Dykam/cfe468875197b7a5d889c68ba8729782

Upon more testing, it disappeared after playing with the disks inserted. Found out it's one specific disk. When it's inserted, the air item appears. Reopening the grid multiplies the item, flooding it if you keep going. Removing gets rid of them all, reinserting shows one air item again.

After swapping the disks a bit the issue disappeared entirely...

tl;dr couldn't reproduce, hope here is enough information to figure it out.

**Comment by Dykam** [comment]
Adding another crashlog https://gist.github.com/Dykam/c37e0f780fe9bfe3262959476f5c5b0c It seems to be reoccuring, but this time it bricked my world. Somehow Itemstack(0, 0) keeps showing up in the system. But I'm not entirely sure how to trace that back to a cause. Though it sounds like something which RS shouldn't even let in even if an external mod tries to insert it.

**Comment by raoulvdberge** [comment]
That is another issue, please make a new issue for that one...

**Comment by Dykam** [comment]
It appears directly related, as both seem involve empty air itemstacks. But I'll split it into another issue.
